person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if is_of_age? && @parent_permission
+
+    false
+  end
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
[OPTIONAL] Since all comparisons always result in true or false and Ruby returns always the result of the last line of any method, you can rewrite what you have like this:

```
de of_age?
  @age >= 18 # It will return false or true
end
```